### ISSUE TITLE
removed old parameter store key/value pairs (after removed from environment helm charts)

### ISSUE THREAD

**Issue by patrick-black-oddball** [department-of-veterans-affairs/va.gov-team#95568]
**Blocked by:** https://github.com/department-of-veterans-affairs/va.gov-team/issues/96548 https://github.com/department-of-veterans-affairs/va.gov-team/issues/95566

After the environment helm charts are updated and old parameter store paths are no longer referenced by the vets-api helm charts, remove the old keys from parameter store.

Care should be taken to not remove parameter store keys used by other applications. As a result, check the cloudTrails access logs to ensure that the key is not being accessed

Over 1000 keys

## Tasks
- [ ] Remove the old parameter store keys from param store

## Acceptance Criteria
- [ ] All unused parameters store keys are removes

**Comment by stevenjcumming** [comment]
Moving to a new ticket
